
<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SingleStock</class>
 <widget class="QWidget" name="SingleStock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,5,1">
     <item>
      <widget class="QLabel" name="stockNameLbl">
       <property name="font">
        <font>
         <family>Sans Serif</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="StockPriceHistoryPlot" name="plot" native="true"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,2,0">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <widget class="QPushButton" name="buyButton">
           <property name="text">
            <string>Buy</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sellButton">
           <property name="text">
            <string>Sell</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,2">
           <item>
            <widget class="QLabel" name="Current_Price_Label">
             <property name="text">
              <string>Price:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdPrice">
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,2">
           <item>
            <widget class="QLabel" name="Stocks_Label">
             <property name="text">
              <string>Shares:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLCDNumber" name="lcdStocks">
             <property name="segmentStyle">
              <enum>QLCDNumber::Flat</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Buy step:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="orderStep">
           <property name="maximum">
            <number>1000000</number>
           </property>
           <property name="singleStep">
            <number>10</number>
           </property>
           <property name="value">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>StockPriceHistoryPlot</class>
   <extends>QWidget</extends>
   <header>stockpricehistoryplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>